Articles/Packages/Extending/index.md

```diff
@@ -0,0 +1,80 @@
+- Solving real life problems
+- Inspiring people on what Umbraco can be made capable of
+
+### Categories of packages
+
+Packages provide a wide variety of functionality, and can often span multiple categories. In general, though, the functionality they provide fall into these main groups:
+
+ - [Schema Extensions](#schema-extensions)
+ - [Management Extensions](#management-extensions)
+ - [Starter Kits](#starter-kits)
+ - [Content Apps](#content-apps)
+ - [Integration Extensions](#integration-extensions)
+
+#### Schema Extensions
+
+A package that can be categorised as a Schema Extension will extend the default Umbraco Schema. Schema in this sense is things like Data Types, Property Editors, Document Types and Media Types. By extending Umbraco with packages such as [Our.Umbraco.GMaps](https://our.umbraco.com/packages/backoffice-extensions/ourumbracogmaps-google-maps-for-umbraco-8/) it gives editors greater capabilities when they are populating their content pages.
+
+#### Management Extensions
+
+A Management Extension package helps you manage your site, and provides info to the users. Management extensions can be made of many different parts, but often they include custom sections or dashboards for you to manage something.
+A good example of a management extension package is [Diplo God Mode](https://our.umbraco.com/packages/developer-tools/diplo-god-mode/) which gives you a lot of extra info and tools to manage your site.
+
+#### Starter Kits
+
+Starter kits are, as the name suggests, a package that helps you set up a starter version of whatever you want to build. Most starter kit packages are for starting a website, and include schema like Document Types and Templates as well as content nodes and media. There are also some quite specialized starter kits, for example for creating a blog. Umbraco HQ has released their [own starter kit](https://our.umbraco.com/packages/starter-kits/the-starter-kit/), that creates a small site with the most commonly used parts. 
+
+#### Content Apps
+
+:::note
+Content Apps are only available from Umbraco 8
+:::
+
+Content apps are almost like dashboards for content nodes that are intended to display node specific information. A good example of this is the [Preflight](https://our.umbraco.com/packages/backoffice-extensions/preflight-content-health-checks-for-umbraco-8/) content app. It shows you readability scores for your written content, directly on each content node.
+
+#### Integration extensions
+
+This type of package can be a lot of things, and can include a lot of the other package types in it. They are generally integrating a larger system into Umbraco. A good example could be an e-commerce package such as [UCommerce](https://our.umbraco.com/packages/website-utilities/ucommerce/), that includes an entire webshop module for Umbraco.
+
+## [Types of Packages](./package-types.md)
+
+There are two common types of package for Umbraco; Package zip files and NuGet Packages.
```

Comment:
```suggestion
There are two common types of package for Umbraco: Package zip files and NuGet Packages.
```

Reply:
I have to be completely honest, I'm not well versed enough in grammar to know if ":" or ";" is correct

Reply:
My daughter is just learning this, so I'm only going by her!  But think `:` should precede a list, `;` is more for separating two parts of a sentence (I think of it as half-way between a `,` and `.`).

Reply:
Ahh yeah I think you're right, I just remembered this old comic that explains it hilariously: https://theoatmeal.com/comics/semicolon